drivers/starburst/src/metabase/driver/implementation/execute.clj

```diff
@@ -168,10 +169,13 @@
 ; Metabase tests require a specific error when an invalid number of parameters are passed
 (defn handle-execution-error
   [e]
-  (log/fatalf (.getMessage e))
-  (if (clojure.string/includes? (.getMessage e) "Incorrect number of parameters")
-    (throw (Exception. "It looks like we got more parameters than we can handle, remember that parameters cannot be used in comments or as identifiers."))
-    (throw e)))
+  (let [message (.getMessage e)]
+    (cond
+      (clojure.string/includes? message "Expecting: 'USING'")
```

Comment:
Can you provide an example of query submitted by the driver that generates this error message?

Reply:
Metabase allows users to execute raw SQL queries, so who knows what could be triggering this message.